.github/workflows/ci.yaml

```diff
@@ -18,3 +18,8 @@ jobs:
               run: bun install
             - name: Run Prettier
               run: bun x prettier --check .
+
+            - name: Upload coverage reports to Codecov
```

Comment:
I think you will need a test runner that outputs coverage? Not sure exactly how it works with bun

Reply:
yeah need to actually complete all the tests...